src/core/loadBlockInterval.ts

```diff
@@ -0,0 +1,22 @@
+import { type } from 'os'
+import blockIntervalPointer from './blockIntervalPointer'
+import storageLoad from './storageLoad'
+import { RpcMethodCall, BigIntish } from '@brinkninja/types'
+
+export type loadBlockIntervalArgs = {
+  signer: string,
+  id: BigIntish
+}
+
+function loadBlockInterval ({
+  signer,
+  id
+}: loadBlockIntervalArgs): RpcMethodCall {
+
+  const pointer  = blockIntervalPointer({ id });
+  if(!pointer) throw new Error(`Invalid pointer for block interval ${id}`)
```

Comment:
maybe throw a try/catch around the blockIntervalPointer call as well? that could potentially fail. for example, what happens if id is null?